README.md

```diff
@@ -19,11 +19,18 @@ The repository is structured into the several crates.
 
 - `core`: Contains the core traits and structs necessary for the testcontainers ecosystem.
 - `cli_client`: Contains an implementation of the `Docker` trait that uses the `docker`-CLI to issue commands.
-- The folder `images` contains several crates named after the respective docker image. Each crate adds support for one particular image. This allows to selectively import the images you need.
+- The folder `images` contains several crates named after the respective docker image. 
+Each crate adds support for one particular image. This allows to selectively import the images you need.
```

Comment:
Randomly add a new line?

Reply:
Wanted to follow the pattern of one sentence per line.

Reply:
But you still have 2 sentences on the line.

Reply:
Good point :D 
Will change that.